### PR TITLE
Add project

### PR DESCRIPTION
**Pull request by TheYisusGithub**

### CODE REVIEW

File: application.properties [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#Setting the count of user name and password.
+user_name=
```
Comment (raulchoque-eng) [inline application.properties:2]: Write some sentence to remember that here we need put values E.g. "user_name=Put user_name"

File: application.properties [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+password_n=
```
Comment (raulchoque-eng) [inline application.properties:3]: Write some sentence to remember that here we need put values E.g. "password_n=Put password_n"

File: application.properties [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+#Set the web Page URL
+url=
```
Comment (raulchoque-eng) [inline application.properties:5]: Write some sentence to remember that here we need put values E.g. "url=Put url"

File: build.gradle [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+version '1.0-SNAPSHOT'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testCompile group: 'junit', name: 'junit', version: '4.12'
```
Comment (raulchoque-eng) [inline build.gradle]: Clear this dependency is not necessary, please.

File: gradle.properties [hunk 5]
```diff
@@ -0,0 +1,4 @@
+browser=
```
Comment (raulchoque-eng) [inline gradle.properties:1]: Put this value, please.

File: gradle.properties [hunk 6]
```diff
@@ -0,0 +1,4 @@
...
+implicit=
```
Comment (raulchoque-eng) [inline gradle.properties:2]: Put this value, please.

File: gradle.properties [hunk 7]
```diff
@@ -0,0 +1,4 @@
...
+explicit=
```
Comment (raulchoque-eng) [inline gradle.properties:3]: Put this value, please.

File: gradle/wrapper/gradle-wrapper.properties [hunk 8]
```diff
@@ -0,0 +1,6 @@
+#Sat Sep 07 12:09:50 BOT 2019
```
Comment (raulchoque-eng) [inline gradle/wrapper/gradle-wrapper.properties:1]: Delete this file please is not necessary.

File: src/main/java/core/config/application.properties [hunk 9]
```diff
@@ -0,0 +1,5 @@
+#Setting the count of user name and password.
```
Comment (raulchoque-eng) [inline src/main/java/core/config/application.properties:1]: Delete this file please you have two application.properties

File: src/main/java/core/selenium/webdrivers/Chrome.java [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+    /**
+     * Get the Chrome org.fundacionjala.core.ui.driver.
+     *
+     * @return the ChromeDriver.
+     */
+    public WebDriver getDriver() {
+        WebDriverManager.chromedriver().setup();
+        return new ChromeDriver();
+    }
+}
```
Comment (raulchoque-eng) [inline src/main/java/core/selenium/webdrivers/Chrome.java:35]: Add a white line bellow this line, please.

File: src/main/java/core/util/ReadProperties.java [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+    private String url;
+    private static ReadProperties ourInstance = new ReadProperties();
+
+    /**
+     * Method return the instance to read properties.
+     *
+     * @return instance to type ReadProperties
+     */
+    public static ReadProperties getInstance() {
+
```
Comment (raulchoque-eng) [inline src/main/java/core/util/ReadProperties.java:38]: Delete this line, please.

File: src/main/java/core/util/ReadProperties.java [hunk 12]
```diff
@@ -0,0 +1,85 @@
...
+    /**
+     * Method let get the url.
+     *
+     * @return url.
+     */
+    public String getUrl() {
+        return url;
+    }
+
```
Comment (raulchoque-eng) [inline src/main/java/core/util/ReadProperties.java:84]: Delete this line, please.

File: src/main/java/kanbanflow/ui/pages/Board.java [hunk 13]
```diff
@@ -0,0 +1,101 @@
...
+public class Board extends BasePage {
+    @FindBy(xpath = "//h2[contains(@title, 'To-do')]/parent::div/following-sibling::button")
+    private WebElement addTaskButton;
+    @FindBy(className = "addTaskDialog-name")
+    private WebElement addTaskName;
+    @FindBy(className = "addTaskDialog-buttonWrapper")
+    private WebElement sendSave;
+    @FindBy(className = "addTaskDialog-wrapper")
+    private WebElement modal;
+
```
Comment (raulchoque-eng) [inline src/main/java/kanbanflow/ui/pages/Board.java]: Delete this line and the bellow line, please.

File: src/main/java/kanbanflow/ui/pages/Board.java [hunk 14]
```diff
@@ -0,0 +1,101 @@
...
+    }
+
+    /**
+     * Verify when send a invitations if the user exists.
+     * @param email the user.
+     * @return compare if member exist.
+     */
+    public boolean verifyIfExist(String email){
+        sendInvite.click();
+        System.out.println(email+" is already a member of the board");
```
Comment (raulchoque-eng) [inline src/main/java/kanbanflow/ui/pages/Board.java]: Delete this print, please.

File: src/main/java/kanbanflow/ui/pages/InviteMember.java [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+
+public class InviteMember extends BasePage {
+    @FindBy(className = "headerMembers-add")
+    private WebElement addMember;
+    @FindBy(className = "chipsInput-input")
+    private WebElement addEmail;
+    @FindBy(className = "inviteBoardMemberDialog-invite button button--success")
+    private WebElement sendInvite;
+
+    public void setAddMember(String email) {
```
Comment (raulchoque-eng) [inline src/main/java/kanbanflow/ui/pages/InviteMember.java]: Comment this method, please.

File: src/main/java/kanbanflow/ui/pages/LoginPage.java [hunk 16]
```diff
@@ -0,0 +1,51 @@
...
+     *
+     * @param email    type string.
+     * @param password type string.
+     */
+    public void signIn(String email, String password) {
+        inputEmail.sendKeys(email);
+        inputPassword.sendKeys(password);
+        submitButton.click();
+    }
+
```
Comment (raulchoque-eng) [inline src/main/java/kanbanflow/ui/pages/LoginPage.java:50]: Delete this line, please

File: src/main/java/kanbanflow/ui/pages/PageTransporter.java [hunk 17]
```diff
@@ -0,0 +1,46 @@
...
+ */
+public class PageTransporter extends BasePage {
+    @FindBy(css = "[href='/login']")
+    private WebElement loginButton;
+
+    /**
+     *Constructor to let set a driver.
+     */
+    public PageTransporter() {
+        System.out.println(ReadProperties.getInstance().getUrl());
```
Comment (raulchoque-eng) [inline src/main/java/kanbanflow/ui/pages/PageTransporter.java:34]: Delete this print, please.

File: src/test/java/kanbanflow/ui/pages/LoginPageTest.java [hunk 18]
```diff
@@ -0,0 +1,27 @@
+//package kanbanflow.ui.pages;
```
Comment (raulchoque-eng) [inline src/test/java/kanbanflow/ui/pages/LoginPageTest.java]: Delete or decomment all these lines, please.

File: src/test/java/runner/ReportGenerator.java [hunk 19]
```diff
@@ -0,0 +1,45 @@
...
+
+    /**
+     * The method generateReport is to set up and create a report.
+     */
+    public static void generateReport() {
+        final File reportOutputDirectory = new File("target");
+        final List<String> jsonFiles = new ArrayList<>();
+        jsonFiles.add("target/cucumber.json");
+        final String projectName = "Salesforce";
+        final boolean runWithJenkins = false;
```
Comment (raulchoque-eng) [inline src/test/java/runner/ReportGenerator.java]: Change the name of project, please.

File: src/test/java/runner/Runner.java [hunk 20]
```diff
@@ -0,0 +1,48 @@
...
+        plugin = {"pretty",
+                "html:target/cucumber-pretty",
+                "json:target/cucumber.json",
+                "rerun:target/rerun.txt"},
+        glue = {"steps"},
+        features = {"src/test/resources/features/"},
+        monochrome = true)
+public class Runner extends AbstractTestNGCucumberTests {
+    @AfterSuite
+    public void closeBrowser() {
```
Comment (raulchoque-eng) [inline src/test/java/runner/Runner.java]: Comment this method, please.

File: src/test/java/steps/KanbanStep.java [hunk 21]
```diff
@@ -0,0 +1,122 @@
...
+    LoginPage loginPage;
+    PageTransporter pageTransporter;
+    InviteMember inviteMember;
+    Board board= new Board();
+
+    /**
+     * Method to compare if the access is successfuly.
+     */
+    @Then("The access is successfuly")
+    public void theAccessIsSuccessfuly() {
```
Comment (raulchoque-eng) [inline src/test/java/steps/KanbanStep.java]: Rename this method please, start with a verb.

File: src/test/java/steps/KanbanStep.java [hunk 22]
```diff
@@ -0,0 +1,122 @@
...
+        board=new Board();
+        Assert.assertEquals(board.isBoardPage(),true);
+        WebDriverManager.getInstance().getDriver().close();
+    }
+
+    /**
+     * Method to let me got to the page kanban flow.
+     */
+    @Given("The user go to the page kanbanflow")
+    public void theUserGoToThePageKanbanflow() {
```
Comment (raulchoque-eng) [inline src/test/java/steps/KanbanStep.java]: Rename this method please, start with a verb.

File: src/test/java/steps/KanbanStep.java [hunk 23]
```diff
@@ -0,0 +1,122 @@
...
+        pageTransporter = new PageTransporter();
+        loginPage = pageTransporter.goToLoginPage();
+    }
+
+    /**
+     * Method to let login to page kanban flow.
+     */
+    @When("The user login to kanbanflow")
+    public void theUserLoginToKanbanflow() {
```
Comment (raulchoque-eng) [inline src/test/java/steps/KanbanStep.java]: Rename this method please, start with a verb.

File: src/test/java/steps/KanbanStep.java [hunk 24]
```diff
@@ -0,0 +1,122 @@
...
+        loginPage.signIn(ReadProperties.getInstance().getUserName(), ReadProperties.getInstance().getPassword());
+    }
+
+    /**
+     * Method to let the user go to the page kanban flow and let login.
+     *
+     */
+    @Given("The user go to the page kanbanflow to login")
+    public void theUserGoToThePageKanbanflowToLogin() {
```
Comment (raulchoque-eng) [inline src/test/java/steps/KanbanStep.java]: Rename this method please, start with a verb.

File: src/test/java/steps/KanbanStep.java [hunk 25]
```diff
@@ -0,0 +1,122 @@
...
+        loginPage = pageTransporter.goToLoginPage();
+        loginPage.signIn(ReadProperties.getInstance().getUserName(), ReadProperties.getInstance().getPassword());
+    }
+
+
+    /**
+     * Method to let add to do task in the page kandan flow.
+     */
+    @When("The user adds to-do task")
+    public void theUserAddsToDoTask() {
```
Comment (raulchoque-eng) [inline src/test/java/steps/KanbanStep.java]: Rename this method please, start with a verb.

File: src/test/java/steps/KanbanStep.java [hunk 26]
```diff
@@ -0,0 +1,122 @@
...
+//        board = new Board();
```
Comment (raulchoque-eng) [inline src/test/java/steps/KanbanStep.java]: Decomment or delete this line, please.

File: src/test/java/steps/KanbanStep.java [hunk 27]
```diff
@@ -0,0 +1,122 @@
...
+        board.setAddNewToDo("Task1");
+    }
+
+    /**
+     * Method to let me compare if the task was create.
+     */
+    @Then("The task was create")
+    public void theTaskWasCreate() {
```
Comment (raulchoque-eng) [inline src/test/java/steps/KanbanStep.java]: Rename this method please, start with a verb.

File: src/test/java/steps/KanbanStep.java [hunk 28]
```diff
@@ -0,0 +1,122 @@
...
+        WebDriverManager.getInstance().getDriver().close();
+    }
+
+
+    @Given("The user add email invitations")
+    public void theUserAddEmailInvitations() {
```
Comment (raulchoque-eng) [inline src/test/java/steps/KanbanStep.java]: Rename this method please, start with a verb.

File: src/test/java/steps/KanbanStep.java [hunk 29]
```diff
@@ -0,0 +1,122 @@
...
+        board.addNewInviteUser();
+    }
+
+    @Then("The page show message when show warning the user exist")
+    public void thePageShowMessageWhenShowWarningTheUserExist() {
```
Comment (raulchoque-eng) [inline src/test/java/steps/KanbanStep.java]: Rename this method please, start with a verb.

File: src/test/java/steps/KanbanStep.java [hunk 30]
```diff
@@ -0,0 +1,122 @@
...
+        Assert.assertEquals(true,board.verifyIfExist("jesus.menacho@fundacion-jala.org"));
+        WebDriverManager.getInstance().getDriver().close();
+    }
+
+    @Then("The invite was send successfuly")
+    public void theInviteWasSendSuccessfuly() {
```
Comment (raulchoque-eng) [inline src/test/java/steps/KanbanStep.java]: Rename this method please, start with a verb.

File: src/test/java/steps/KanbanStep.java [hunk 31]
```diff
@@ -0,0 +1,122 @@
...
+      //  WebDriverManager.getInstance().getDriver().close();
```
Comment (raulchoque-eng) [inline src/test/java/steps/KanbanStep.java]: Decomment or delete this line, please.

File: src/test/java/steps/KanbanStep.java [hunk 32]
```diff
@@ -0,0 +1,122 @@
...
+    }
+
+    @When("The user sent email invitations")
+    public void theUserSentEmailInvitations() {
```
Comment (raulchoque-eng) [inline src/test/java/steps/KanbanStep.java]: Rename this method please, start with a verb.

File: src/test/java/steps/KanbanStep.java [hunk 33]
```diff
@@ -0,0 +1,122 @@
...
+        board.writeEmailInvitations("jesus.menacho@fundacion-jala.org");
+    }
+
+    /**
+     * Method add the new user.
+     */
+    @When("The user add new member")
+    public void theUserAddNewMember() {
```
Comment (raulchoque-eng) [inline src/test/java/steps/KanbanStep.java]: Rename this method please, start with a verb.

File: src/test/resources/features/AddTask.feature [hunk 34]
```diff
@@ -0,0 +1,24 @@
+Feature: Creation funtional and acceptance
+  @Acceptance
+  Scenario: The user adds new task
+    Given The user logs in to the kanbanflow page
+    When The user adds new to-do task
+    Then The task was create
+    And show the task in the board
+
+
```
Comment (raulchoque-eng) [inline src/test/resources/features/AddTask.feature:8]: Delete this and bellow white lines, please.

File: src/main/java/kanbanflow/ui/pages/LoginPage.java [hunk 35]
```diff
@@ -0,0 +1,51 @@
...
+    public LoginPage() {
+    }
+
+    /**
+     * Method to let me login in the page kanban flow.
+     *
+     * @param email    type string.
+     * @param password type string.
+     */
+    public void signIn(String email, String password) {
```
Comment (raulchoque-eng) [inline src/main/java/kanbanflow/ui/pages/LoginPage.java:45]: Add final to these parameters email and password, please.

File: application.properties (hunk 1 above)
Comment (TheYisusGithub) [reply application.properties:2]: Please in the next review a correct that.

File: application.properties (hunk 2 above)
Comment (TheYisusGithub) [reply application.properties:3]: Please in the review a correct that.

File: application.properties (hunk 1 above)
Comment (TheYisusGithub) [reply application.properties:2]: Please dear rail this is a step that a considerate not relevant but in the next review a going implement that.

File: application.properties (hunk 3 above)
Comment (TheYisusGithub) [reply application.properties:5]: Please dear rail this is a step that a considerate not relevant but in the next review a going implement that.

File: application.properties (hunk 2 above)
Comment (TheYisusGithub) [reply application.properties:3]: Please dear rail this is a step that a considerate not relevant but in the next review a going implement that.

**Pull request merged**